buildSrc/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -59,6 +64,9 @@ extraJavaModuleInfo {
     automaticModule("io.prometheus:simpleclient_httpserver", "io.prometheus.simpleclient.httpserver")
 
     automaticModule("j2objc-annotations-1.3.jar", "j2objc.annotations")
+    automaticModule("io.perfmark:perfmark-api", "perfmark.api")
+
+    automaticModule("org.eclipse.microprofile.health:microprofile-health-api", "microprofile.health.api")
```

Comment:
ARHG! Unbelievable that such project does not have at least an automatic module name. Created an issue: https://github.com/eclipse/microprofile-health/issues/322